rpcs3/rpcs3qt/debugger_frame.cpp

```diff
@@ -1156,6 +1156,20 @@ void debugger_frame::DoStep(bool step_over)
 			m_debugger_list->ShowAddress(cpu->get_pc() + 4, false);
 		}
 
+		if (step_over && cpu->id_type() == 0x55)
+		{
+			const bool was_paused = cpu->is_paused();
+			static_cast<rsx::thread*>(cpu)->pause_on_draw = true;
+
+			if (was_paused)
+			{
+				m_btn_run->click();
```

Comment:
this is not maintainable.
transform the lambda to a slot and use it as a function here

Reply:
Fixed.